panda.py

```diff
@@ -0,0 +1,224 @@
+from pxr import Usd, Vt
+import json
+
+data = {}
+stage = Usd.Stage.Open('franka.usda')
+panda = stage.GetPrimAtPath('/panda')
+
+link0 = stage.GetPrimAtPath('/panda/panda_link0')
```

Comment:
In the next iteration or now, we may with to understand the DOF of the robot and have some assumptions about it.

We could have a for loop here that adds entries to a dictionary or list of links and joints.
```python
links = []
joints = []
for i in range(0, n_dof):
  link_name = "/panda/panda_link"+str(i)"
  joint_name = link_name+"/panda_joint"+str(i+1)
  links.append(stage.GetPrimAtPath(link_name))
  joints.append(stage.GetPrimAtPath(joint_name))
```